doc/pages/administrator/030-installation_kubernetes.md

```diff
@@ -19,23 +22,66 @@ version to install - when not specified, the latest version will be installed in
+if you are dealing with at least v1.0.
+
+To upgrade the Operator, use the dedicated `helm upgrade` command:
 
 ```bash
-astartectl cluster upgrade-operator
+helm upgrade astarte-operator astarte/astarte-operator [--version <new-version>]
 ```
 
-Just like the `install-operator` command, the `--version` switch allows to specify the version to upgrade to -
-when not specified, the latest version will be fetched and used.
+The optional `--version` switch allows to specify the version to upgrade to - when not specified, the latest
+version will be fetched and used.
+
+Currently, only the first version of the chart is available within `helm.astarte-platform.org` and the `helm upgrade`
+operation will lead to an error. Bear with that as long as new versions will be released.
+
+### Upgrading the Operator (from v0.11 to v1.0)
+
+The upgrade procedure from v0.11 to v1.0 may be thought of as non-linear procedure as the deployment and handling of
+the Operator's lifecylcle has changed: if v0.11 is entirely handled with `astartectl`, v1.0 employs Helm charts.
+
+To ensure a consistent upgrade procedure, the following steps are required:
+
++ Remove the Operator's Service Account, Cluster Roles and Cluster Role Bindings:
+```bash
+kubectl delete serviceaccounts -n kube-system astarte-operator
+kubectl delete clusterroles.rbac.authorization.k8s.io astarte-operator
+kubectl delete clusterrolebindings.rbac.authorization.k8s.io astarte-operator
+```
+
++ Delete the Operator's deployment:
+```bash
+kubectl delete deployments.app -n kube-system astarte-operator
+```
+
+**DO NOT** delete Astarte's CRDs! This will lead to the deletion of the entire astarte deployment with a consequent
+data loss.
+
+To restore the Operator's functionalities with v1.0, all you have to do is following the install instructions, i.e.:
+```bash
+helm repo add astarte https://helm.astarte-platform.org
+helm repo update
+helm install astarte-operator astarte/astarte-operator --version 1.0.0-alpha.1
+```
 
 ## Uninstalling the Operator
 
-To uninstall the Operator, use the dedicated `uninstall-operator` command.
+To uninstall the Operator, use the dedicated `helm uninstall` command.
 
 ```bash
-astartectl cluster uninstall-operator
+helm uninstall astarte-operator
 ```
 
-This command will refuse to execute if any Astarte instances are active in the Cluster. Uninstalling the Operator
-is possible only after all managed instances have been deleted too.
-When uninstalling the Operator, all resources installed with `install-operator` will be erased completely.
+Uninstalling the Operator can be performed even if an Astarte instance is still active in the cluster. The result of this
+operation consists in the deletion of RBACs and of the operator's deployment itself.
+
+*Note: CRDs are not affected by the uninstall procedure as, according to Helm's choices, forcing their removal may lead to unwanted and (in some cases)
```

Comment:
This line is a little long, ideally we should try to keep markdown to 100 columns